### PR TITLE
[nyc-2019] copy from 2018, removed some old stuff

### PR DESCRIPTION
**Pull request by russAIC**
Initial PR for nyc 2019 dod. Took 2018 nyc documents and cleaned them to be ready for next year.

### CODE REVIEW

File: content/events/2019-new-york-city/propose.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
++++
+Title = "Propose"
+Type = "event"
+Description = "Propose a talk for devopsdays New York City 2019"
++++
+  {{< cfp_dates >}}
+
+<center>
+NOTE: Due to circumstances beyond our control, the talk proposal <b>acceptance emails will be delayed to Dec 7th.</b> Thank you for your patience.
+</center>
```
Comment (bridgetkromhout) [inline content/events/2019-new-york-city/propose.md]: You probably want to remove this note

File: content/events/2019-new-york-city/propose.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <li>Detail is good... but not as important as explaining why your proposal would be interesting</li>
+  <li>Propose your own talk; don't have someone else do it for you.</li>
+  <li>Nominations welcome... if you know someone who has content/experience relevant to the DevOps conversation, please point us in their direction!</li>
+  <li>Multiple proposals welcome... just follow the other rules</li>
+</ul>
+
+<hr>
+
+<strong>How to submit a proposal:</strong>
+Visit our [CFP Site](https://www.papercall.io/cfps/709/submissions/new) and submit a proposal there.
```
Comment (bridgetkromhout) [inline content/events/2019-new-york-city/propose.md]: This is the link to the old CFP site

File: content/events/2019-new-york-city/welcome.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    {{< event_location >}}
+  </div>
+</div>
+
+<div class = "row">
+  <div class = "col-md-2">
+    <strong>Register</strong>
+  </div>
+  <div class = "col-md-8">
+    <a href="https://www.eventbrite.com/e/devopsdays-nyc-2019-tickets-39330760363">Register to attend the conference!</a>
```
Comment (bridgetkromhout) [inline content/events/2019-new-york-city/welcome.md]: This is the old eventbrite link

File: content/events/2019-new-york-city/welcome.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+</div>
+
+<div class = "row">
+  <div class = "col-md-2">
+    <strong>Speakers</strong>
+  </div>
+  <div class = "col-md-8">
+    Check out the {{< event_link page="speakers" text="speakers!" >}}
+  </div>
+</div>
```
Comment (bridgetkromhout) [inline content/events/2019-new-york-city/welcome.md]: You'll want to comment out the program and speakers stuff for now.

**Comment by bridgetkromhout** [comment]
Okay, so we won't merge this without:
* the yml file that configures the event
* getting the organizer email list and Slack set up

You probably would be better off just using the new event script instead of copying part of 2018:
https://github.com/devopsdays/devopsdays-web/blob/master/utilities/README.md#events

But if you want to do it manually, please make sure you're not missing the yml file; you need it.

**Comment by russAIC** [comment]
Thank you for the suggestion, opening new pr now after using the shell script :)

**Pull request closed without merging**